tests/Response/ParticipantResponseTest.php

```diff
@@ -31,7 +31,13 @@ public function testRequest()
 
         $client = new RewardStack\Client($authProxy);
 
-        $participantRequest = new Participant\ParticipantRequest('TESTPARTICIPANT1');
+        $participantRequest = new Participant\ParticipantRequest(
+            'sharecare',
+            'TESTPARTICIPANT1',
+            'Zech',
+            'Walden2',
```

Comment:
Maybe we change this to technology@alldigitalrewards.com or development@alldigitalrewards.com